doc/_utils/redirects.py

```diff
@@ -194,10 +194,14 @@
     ("device_guides/working_with_nrf/nrf70/nrf7002eb_gs", "app_dev/device_guides/nrf70/nrf7002eb_dev_guide"), # Developing with nRF7002 EB
     ("device_guides/nrf70/nrf7002eb_dev_guide", "app_dev/device_guides/nrf70/nrf7002eb_dev_guide"),
     ("device_guides/nrf54l", "app_dev/device_guides/nrf54l"), # Developing with nRF54L Series
-    ("device_guides/working_with_nrf/nrf54l/features", "app_dev/device_guides/working_with_nrf/nrf54l/features"), # Features of the nRF54L15 PDK
-    ("device_guides/working_with_nrf/nrf54l/nrf54l15_gs", "app_dev/device_guides/working_with_nrf/nrf54l/nrf54l15_gs"), # Getting started with nRF54L15 PDK
-    ("device_guides/working_with_nrf/nrf54l/testing_dfu", "app_dev/device_guides/working_with_nrf/nrf54l/testing_dfu"), # Testing the DFU solution
-    ("device_guides/working_with_nrf/nrf54l/peripheral_sensor_node_shield", "app_dev/device_guides/working_with_nrf/nrf54l/peripheral_sensor_node_shield"), # Developing with Peripheral Sensor node shield
+    ("device_guides/working_with_nrf/nrf54l/features", "app_dev/device_guides/nrf54l/features"), # Features of the nRF54L15 PDK
+    ("app_dev/device_guides/working_with_nrf/nrf54l/features", "app_dev/device_guides/nrf54l/features"),
+    ("device_guides/working_with_nrf/nrf54l/nrf54l15_gs", "app_dev/device_guides/nrf54l/nrf54l15_gs"), # Getting started with nRF54L15 PDK
```

Comment:
This redirection is not working. Also, I do not see the file app_dev/device_guides/nrf54l/nrf54l15_gs.html.